### DISCUSSION TITLE
Flutter package google_sign_in for user authentication in Pocketbase

### DISCUSSION THREAD

**Discussion by suelitonx** [pocketbase/pocketbase#4714] [Q&A] [answered]
I would like to know if there is a possibility of using the Flutter package [google_sign_in](https://pub.dev/packages/google_sign_in) and authenticating the user in Pocketbase.
The advantage of using the package is having the "native" connection, without needing to use the url_launcher and opening the user's browser.

![Screenshot_6](https://github.com/pocketbase/pocketbase/assets/145073364/7d8c77c1-6bc0-4c24-bddc-0e0eae470d5e)

![image](https://github.com/pocketbase/pocketbase/assets/145073364/77f64bc2-f58a-402c-8f6e-6e67f6713dfc)

**Comment by ganigeorgiev** [comment, marked as answer]
It is not supported (at least not out of the box).

Note that you don't necessary have to use the `url_launcher` package. If you want to make the transition between the browser and the app more seamless you can try [Custom tabs](https://developer.chrome.com/docs/android/custom-tabs/) for example with [`flutter_custom_tabs`](https://pub.dev/packages/flutter_custom_tabs).
